.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,27 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: clone the repo
+      run: git clone https://github.com/sumangala028/learn-github-action.git
+    - name: execute the react project
+      run : npm start
```

Comment:
what does `npm start`?

Reply:
To start the react project

Reply:
Do you need to start project here? Check other scripts

Reply:
I don't have a clear view of this github actions

Reply:
`npm start` is not a github action :D. It's just a CMD script. 

`npm start` - This runs an arbitrary command specified in the package’s "start" property of its "scripts" object. 

So what are other scripts you have?

Reply:
Other scripts means?
What?

Reply:
How `npm start` starts your react application? Is it a miracle? 

It will check for the available scripts in your package.json and execute the actions for that name. https://github.com/sumangala028/learn-github-action/blob/master/hello-world/package.json#L11

Reply:
if package.json is like this, 
```
{
  "name": "hello-world",
  "version": "0.1.0",
  "private": true,
  "dependencies": {
    "react": "^16.13.1",
    "react-dom": "^16.13.1",
    "react-scripts": "0.9.5"
  },
  "devDependencies": {},
  "scripts": {
    "sumi-idiot": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test --env=jsdom",
    "eject": "react-scripts eject"
  }
}

```
then `npm sumi-idiot` will start your application

Reply:
The package.json is like this, 

```
{
  "name": "hello-world",
  "version": "0.1.0",
  "private": true,
  "dependencies": {
    "react": "^16.13.1",
    "react-dom": "^16.13.1",
    "react-scripts": "0.9.5"
  },
  "devDependencies": {},
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test --env=jsdom",
    "eject": "react-scripts eject"
  }
}
```

Reply:
So what are other script you can run?

Reply:
which scripts?

Reply:
`npm start` is a script. Like that what are other scripts?

Reply:
```
npm build
npm test
```

Like these?